### PR TITLE
Remove duplicated workflow

### PR DESCRIPTION
**Pull request by chuckwagoncomputing**
The workflows for console with Java version 8 and the one intended for Java 11 are identical. Might as well get rid of the 11 version for now.

### CODE REVIEW

File: firmware/controllers/date_stamp.h [hunk 1]
```diff
@@ -1,3 +1,2 @@
-#ifndef VCS_DATE
-#define VCS_DATE 20200617
-#endif
\ No newline at end of file
+#pragma once
+#define VCS_DATE 20200618
```
Comment (mck1117) [inline firmware/controllers/date_stamp.h:2]: is this change on purpose?

Comment (chuckwagoncomputing) [reply firmware/controllers/date_stamp.h:2]: nope. set_date ran and I didn't notice.

Comment (chuckwagoncomputing) [reply firmware/controllers/date_stamp.h:2]: Won't hurt a thing though.

Comment (mck1117) [reply firmware/controllers/date_stamp.h:2]: did it run on your branch? perhaps it should only run when on real rusefi/master

Comment (mck1117) [reply firmware/controllers/date_stamp.h:2]: yep looks like that's exactly what happened

Comment (chuckwagoncomputing) [reply firmware/controllers/date_stamp.h:2]: good idea. I'll take care of that.
